airbyte-workers/src/main/java/io/airbyte/workers/temporal/exception/DeletedWorkflowException.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.exception;
+
+public class DeletedWorkflowException extends Exception {
```

Comment:
maybe helpful for this exception and the other one to add a javadoc comment explaining what they mean and why they might occur.